operator/src/test/java/org/keycloak/operator/testsuite/integration/Profiles.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.operator.testsuite.integration;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+import java.util.Map;
+
+public class Profiles {
+
+    public static class NoOperator implements QuarkusTestProfile {
```

Comment:
I would not introduce Quarkus Test Profile as we don't really aim for overriding Quarkus Options, IMHO. See my comment above.

Also, not sure why but it appears the `CRDTest` is not executed in the CI with  this profile.